src/components/UI/ResourcesLink/ResourcesLink.test.tsx

```diff
@@ -6,11 +6,13 @@ import { ResourcesLink } from "./ResourcesLink";
 const mockInfo = {
   title: "Source code",
   type: "link",
-  details: {
-    description: "View TradeTrust source code",
-    url: "https://github.com/TradeTrust/tradetrust-website",
-    icon: "/static/images/Github_Octocat.png",
-  },
+  details: [
+    {
+      description: "View TradeTrust source code",
+      url: "https://github.com/TradeTrust/tradetrust-website",
+    },
+  ],
```

Comment:
now that `details` became an array, u want to include 1 more test to say expect 2 items length if mock has 2?

Reply:
Sure! I'll add that:)